aws/resource_aws_docdb_cluster_parameter_group_test.go

```diff
@@ -47,6 +47,35 @@ func TestAccAWSDocDBClusterParameterGroup_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSDocDBClusterParameterGroup_systemParameter(t *testing.T) {
+	var v docdb.DBClusterParameterGroup
+	resourceName := "aws_docdb_cluster_parameter_group.bar"
+
+	parameterGroupName := fmt.Sprintf("cluster-parameter-group-test-terraform-%d", acctest.RandInt())
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		ErrorCheck:   testAccErrorCheck(t, docdb.EndpointsID),
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSDocDBClusterParameterGroupDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSDocDBClusterParameterGroupConfig_SystemParameter(parameterGroupName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSDocDBClusterParameterGroupExists(resourceName, &v),
+					testAccCheckAWSDocDBClusterParameterGroupAttributes(&v, parameterGroupName),
+					testAccMatchResourceAttrRegionalARN(resourceName, "arn", "rds", regexp.MustCompile(fmt.Sprintf("cluster-pg:%s$", parameterGroupName))),
+					resource.TestCheckResourceAttr(resourceName, "name", parameterGroupName),
+					resource.TestCheckResourceAttr(resourceName, "family", "docdb3.6"),
+					resource.TestCheckResourceAttr(resourceName, "description", "Managed by Terraform"),
+					resource.TestCheckResourceAttr(resourceName, "parameter.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "tags.%", "0"),
+				),
+			},
```

Comment:
(Adding `ImportStateVerifyIgnore: []string{"parameter"}`, if necessary)

```suggestion
			},
			{
				ResourceName:      resourceName,
				ImportState:       true,
				ImportStateVerify: true,
			},
```